swayidle.1.scd

```diff
@@ -69,7 +69,7 @@ command line and in the config file.
 	session after <timeout> seconds. Adding an idlehint event will also cause
 	swayidle to call SetIdleHint(false) when run, on resume, unlock, etc.
 
-All commands are executed in a shell.
+All commands are executed in a shell and must be quoted as they are parsed as a single argument
```

Comment:
There is no requirement to quote commands in general, but commands *with whitespaces* need to be quoted to be parsed correctly as positional arguments. You don't need quotes to just run a script without arguments for example.

Reply:
Fair point. I'll update it.